webapp/src/pages/PerformanceAnalysis.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Typography } from "@mui/material";
+import Description from "components/Description";
+import PerformanceAnalysisComparisonTable from "components/Metrics/PerformanceAnalysisTable";
```

Comment:
We normally keep the same name:
```suggestion
import PerformanceAnalysisTable from "components/Metrics/PerformanceAnalysisTable";
```

Reply:
Ohh!... that's a typo. You are right! It should be changed to "PerformanceAnalysisTable."